uiowa/uiowa
# max_input_vars patch

Resolves https://github.com/uiowa/uiowa/issues/6176

# How to test


1. Follow this to set DDEV to 128: https://ddev.readthedocs.io/en/latest/users/extend/customization-extendibility/#custom-php-configuration-phpini
2. blt ds --site=engineering.uiowa.edu
3. Masquerade as a webmaster on Engineering.
4. Go to https://engineering.uiowa.ddev.site/admin/structure/menu/manage/main
5. Make changes to the menu and see if you can save it.
6. Did the changes stick?
7. Follow the same steps on a few other sites with large menus: Tippie is one.
